doc/nle/source/getting_started.rst

```diff
@@ -4,31 +4,32 @@ Getting Started
 Dependencies
 ************
 
-NLE requires `python>=3.7`, `cmake>=3.14` and some NetHack dependencies
+NLE requires `python>=3.8`, `cmake>=3.14` and some NetHack dependencies
```

Comment:
Not the fault of this PR but it says `cmake>=3.15` above. Let's perhaps just drop the version specification in this file?

Reply:
I'd rather have the documentation be consistent with the underlying specification. Right now, `pyproject.toml` says `cmake>=3.18` so I've updated both `README.md` and `getting_started.rst` to match (new commit).

Reply:
If you prefer that, fine. I feel it's better to keep the number of places in the docs that need to be updated for this small, hence to remove it from one location. Otherwise things more easily get out of sync as they did here.